BankingApplication/src/main/java/com/cognologix/bankingApplication/exceptions/CustomerNotFoundExceptionHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.cognologix.bankingApplication.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@RestControllerAdvice
+public class CustomerNotFoundExceptionHandler {
+
+    @ExceptionHandler(CustomerNotFoundException.class)
+    public ResponseEntity<String> hanleCustomerNotFoundException(Exception ex){
+        return new ResponseEntity<>("Exception : "+ex.getMessage(), HttpStatus.BAD_REQUEST);
+    }
```

Comment:
Can we have one Exception Handler class per entity ?
all the exceptions can be caught in them accordingly